mcs/class/Mono.Cairo/Mono.Cairo/SurfaceType.cs

```diff
@@ -44,5 +45,18 @@ public enum SurfaceType
 		BeOS,
 		DirectFB,
 		Svg,
+		OS2,
+		Win32Printing,
+		QuartzImage,
+		Script,
+		Qt,
+		Recording,
+		VG,
+		GL,
+		Drm,
+		Tee,
+		Xml,
+		Skia,
+		SubSurface
```

Comment:
That's `cairo_surface_type` enum from `src/cairo.h` in the cairo source tree, I'm on the master branch of Cairo. Don't merge if no one else show interest in having GL surface bindings. It's not actively developped.